CHANGES/942.misc.rst

```diff
@@ -0,0 +1 @@
+Official support for Python 3.12.
```

Comment:
We write _(most of the)_ individual change note fragments in the past tense, because they're targeted at the end-users and don't necessarily contain information about internal implementation details or decision.
Changelog is supposed to describe what changed compared to the previous version.

Individual fragments should have the same style. When looking at individual PRs on GitHub, it's unobvious and is hard to judge whether it “fits”.

However, we have RTD PR builds enabled, so it's possible to preview at https://yarl--942.org.readthedocs.build/en/942/changes.html whether a fragment contributes to the changelog being cohesive.

I think, this phrase can be rephrased to describe more accurately what the PR does, with the end-users being the target audience in mind.
```suggestion
Declared Python 3.12 supported officially in the distribution package metadata
-- by :user:`edgarrmondragon`.
```